readthedocs/audit/serializers.py

```diff
@@ -0,0 +1,75 @@
+"""
+Serializers used to save the corresponding data from a model in a log entry.
+
+.. note::
+
+   These aren't used in an API, but to extract data from objects to
+   save it in log entries (AuditLog.data).
+"""
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+from readthedocs.invitations.models import Invitation
+from readthedocs.organizations.models import Organization, Team
+from readthedocs.projects.models import Project
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ["id", "username"]
+
+
+class OrganizationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Organization
+        fields = ["id", "slug"]
+
+
+class TeamSerializer(serializers.ModelSerializer):
+
+    organization = OrganizationSerializer()
+
+    class Meta:
+        model = Team
+        fields = ["id", "slug", "organization"]
+
+
+class ProjectSerializer(serializers.ModelSerializer):
+
+    organization = OrganizationSerializer()
+
+    class Meta:
+        model = Project
+        fields = ["id", "slug", "organization"]
+
+
+class InvitationSerializer(serializers.ModelSerializer):
+
+    """Invitation serializer."""
+
+    to_user = UserSerializer()
+    from_user = UserSerializer()
+    object = serializers.SerializerMethodField()
```

Comment:
Isnt' it a problem to override the `object` reserved word?

Reply:
only when used as variable name, we can change it to `obj` if we want